dist/translations/fr.json

```diff
@@ -0,0 +1,77 @@
+{
```

Comment:
The translation file seems to work if it is moved to `static/translations/` instead of `dist/translations`.

Reply:
Strange, was working for me locally ... done

Reply:
Files in `dist` are generated on release and should never be edited manually. We should clarify this in the readme.

Reply:
Yes you're right ... dist is obviously meant to gather the 'end result'. I guess my last commit just was totally against those guidelines ! Let me know if you want a different commit or something.